### PR TITLE
TX-346: Add Installer for JC Remote Assist

### PR DESCRIPTION
**Pull request by wwaughjc**
## Issues
* [TX-346](https://jumpcloud.atlassian.net/browse/TX-346) - Add mac install script for remote assist

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
```
Comment (jburgessjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: staging url likely won't work with VPN?

Comment (sthurstonjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: I notice this points to a `awsstg` URL, but customer can't reach that can they?

Comment (wwaughjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: Good questions, but I'm not sure the answer. @asinghjc could you help out here?

Comment (asinghjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: Hi Scott and Wes,
The awsstg link for downloading the installable files (jumpcloud-assist-app.dmg) are supposed be a public URL. The download links are generated by deploy team with this requirement.
I tried downloading the file on my personal machine using this link and it worked fine.

Thanks,
Ajay

Comment (sthurstonjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: you mean *without*, right Jay?

Comment (jburgessjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: yeah, it was a long day yesterday. edited original message for clarity

Comment (wwaughjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: Unfortunately I still lack an understanding of whether or not this URL is acceptable...

File: PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Remote Assist.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+#### Description
+
+This command will download and install the JumpCloud Remote Assist app on a Windows device.
+
+### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'TODO'
```
Comment (ttatarevicjc) [inline PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Remote Assist.md:51]: https://git.io no longer accepting links to shorten :/ I think we want ...

`Import-JCCommand -URL 'https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Install%20Remote%20Assist.md'`

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#### Description
+
+Downloads and installs the JumpCloud Remote Assist app on a Mac system.
+
+#### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'TODO'
```
Comment (ttatarevicjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:57]: https://git.io/ no longer accepting links to shorten :/ I think we want ...

`Import-JCCommand -URL 'https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Remote%20Assist.md`

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
```
Comment (sthurstonjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md:12]: One other suggestion for these scripts: If we assume that the path will be constant the host will change, you could separate this into fields:
```
DOWNLOAD_HOST="cdn.awsstg.jumpcloud.com"
DMG_PATH="TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
REMOTE_DMG_URL="https://${DOWNLOAD_HOST}/${DMG_PATH}"
```
That might make it a little easier to update later.

**Comment by wwaughjc** [comment]
This PR is being split into two different PRs for better project tracking:
* https://github.com/TheJumpCloud/support/pull/405
* https://github.com/TheJumpCloud/support/pull/406

**Pull request closed without merging**